packages/components/src/components/step/step.stories.ts

```diff
@@ -1,16 +1,28 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-step');
 const { overrideArgs } = storybookHelpers('sd-step');
 const { generateTemplate } = storybookTemplate('sd-step');
 
+/**
+ * Used to determine a step in a process or task.
+ *
+ * Use the `default` slot to set a description for the step. Alternatively, you can use the `description` attribute.
+ * Use the `circle-content` slot to add a content-icon in a non-interactive step.
+ *
+ * **Related components:**
+ * - [sd-step-group](?path=/docs/components-sd-step-group--docs)
+ *
+ * **Related templates:**
+ * - [sd-step-group](?path=/docs/templates-sd-step-group--docs)
+ *
+ */
 export default {
   title: 'Components/sd-step',
+  tags: ['!dev'],
   component: 'sd-step',
   args: overrideArgs([
     {
```

Comment:
Could you show

![image](https://github.com/user-attachments/assets/e33a06e0-be9b-4214-a1b0-1db14c057439)